ManatiCR/fdi-front
# FDI-46 directory page

[FDI-46](https://manati.atlassian.net/browse/FDI-46)
-
**Steps to test**
- Ensure the backend is running
- On the back-end side create several directorio nodes, as many as you want at least 9 (to see the design)
- Go to http://localhost:8080/directorio
- There you should be able to see all the directorio nodes and be able to filter them
